pallets/bridge/src/lib.rs

```diff
@@ -185,6 +201,18 @@ pub mod pallet {
         /// [bridge_token_id, fee]
         BridgeTokenFeeChanged(CurrencyId, BalanceOf<T>),
 
+        /// The status of the bridge token has changed
+        /// [bridge_token_id, enabled]
+        BridgeTokenStatusChanged(CurrencyId, bool),
```

Comment:
Just wondering if we should use the same word in different place

Changed vs Updated

Reply:
yep